pkg/investigations/cpd/cpd.go

```diff
@@ -65,14 +65,16 @@ func (c *Investigation) Run(r *investigation.Resources) (investigation.Investiga
 				logging.Error(err)
 			}
 			if !isValid {
-				if err := r.OcmClient.PostServiceLog(r.Cluster.ID(), byovpcRoutingSL); err != nil {
+				if err := r.OcmClient.PostServiceLog(r.Cluster.ID(), byovpcRoutingSL, c.InformingMode()); err != nil {
```

Comment:
thought: Can we somehow, make the investigation code agnostic to the informing mode?
it would be cool to avoid adding the condition L75-77 below.

I like the extra parameter we pass to functions in the ocm package here, https://github.com/openshift/configuration-anomaly-detection/pull/432/files#diff-83e5cc2dfa3554ded718960969e8a817d6791f7e3c9c769a1136e6a0a3f94820R211 
but suspect we do not want to add this logic to the ocm package

something along the lines of having a layer where the note posting to PD and actions taken are evaluated against the informing mode.

Reply:
I am not too sure however if this approach is the best idea

How would we deal with changes to the investigation? 
I dont think we would want to fallback to informing mode for the whole investigation, but if the newly added condition is hit, would we like SRE to verify?